build.go

```diff
@@ -22,7 +23,7 @@ func (ws *workspaceBuilder) PrepareWorkspace(basename string) (string, error) {
 		return "", fmt.Errorf("failed to create workspace directory: %v", err)
 	}
 
-	_, err = cloneRepo(ws.token, workspacePath, ws.clonePath, ws.owner, ws.repo, ws.ref)
+	_, err = cloneRepo(ws.token, workspacePath, ws.clonePath, ws.owner, ws.repo, ws.ref, ws.timeOut)
```

Comment:
I am positive Go lint persuaded me to do that, but changing should be trivial, will see if I can improve one test that I created.